lib/api/core/hotelClerk.js

```diff
@@ -604,7 +604,7 @@ function subscribeToRoom (roomId, request) {
           m: request.input.metadata
         }};
 
-      if (!this.customers[request.context.connectionId] || !this.customers[request.context.connectionId][roomId]) {
+      if (!this.customers[request.context.connectionId] || this.customers[request.context.connectionId][roomId] === undefined) {
```

Comment:
can `this.customers[request.context.connectionId]` be falsy and not `undefined` ?
I mean: can we write this here: 
```javascript
if (this.customers[request.context.connectionId] === undefined || this.customers[request.context.connectionId][roomId] === undefined) { 
```
?